{"name": "test_clickNewPtsd", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "@pytest.fixture(scope=\"class\")\n    def into_create_ptsd_page():\n        global driver\n        # 前置操作\n        driver = webdriver.Chrome()\n        driver.maximize_window()\n        driver.get(CD.web_login_url)\n        LoginPage(driver).login(LD.success_data[\"用户名\"], LD.success_data[\"密码\"])\n        IndexPage(driver).click_ptsd()\n>       CreatePtsd(driver).is_Exist_baseinfo()\n\nTestCases\\conftest.py:86: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nPageObjects\\createPtsd_p.py:16: in is_Exist_baseinfo\n    self.wait_eleVisible(loc.title,doc=doc)\nCommon\\basepage.py:42: in wait_eleVisible\n    self.save_screenshot(doc)\nCommon\\basepage.py:33: in wait_eleVisible\n    WebDriverWait(self.driver, wait_times, poll_frequency).until(EC.visibility_of_element_located((locator[1])))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"063ec3aae874d917ebcea410c0425e89\")>\nmethod = <selenium.webdriver.support.expected_conditions.visibility_of_element_located object at 0x000001B508257A48>\nmessage = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nc:\\users\\yiii\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "attachments": [{"name": "log", "source": "4b98536e-5a98-4061-97e9-ef0a0bbbb952-attachment.txt", "type": "text/plain"}], "start": 1587371213587, "stop": 1587371213587, "uuid": "635f7a9f-c7ec-4c0c-8e18-0216becc3b0e", "historyId": "aaf4de9910bd93e486b75ff964c903bb", "testCaseId": "0e61401d49a5da7fca97140e7378001d", "fullName": "TestCases.test_02_ptsdList.TestPtsdPage#test_clickNewPtsd", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_02_ptsdList"}, {"name": "subSuite", "value": "TestPtsdPage"}, {"name": "host", "value": "DESKTOP-O2A05I4"}, {"name": "thread", "value": "15584-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_02_ptsdList"}]}